### PR TITLE
Additional language for conformance statement

### PR DESCRIPTION
**Pull request by mrunalp**
Proposed additional conformance language to support future certification work (cribbed from https://tools.ietf.org/html/rfc2616).

Closes #373 

Signed-off-by: Stephen R. Walli <stephen.walli@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
@vbatts @crosbymichael PTAL

File: README.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ Table of Contents
 
 In the specifications in the above table of contents, the keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119) (Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, March 1997).
 
+An implementation is not compliant if it fails to satisfy one or more of the MUST or REQUIRED requirements for the protocols it implements. 
+An implementation that satisfies all the MUST or REQUIRED and all the SHOULD requirements for its protocols is said to be "unconditionally compliant."
```
Comment (jonboulle) [inline README.md]: s/."/"./

**Comment by vishh** [comment]
LGTM except for https://github.com/opencontainers/runtime-spec/pull/374#discussion_r58956137

**Comment by vbatts** [comment]
LGTM save Jon's comment

**Comment by mrunalp** [comment]
Fixed.

**Pull request merged**